packages/website/src/bin/page-generator/test.js

```diff
@@ -44,6 +44,20 @@ async function runGeneratePages({
   );
 }
 
+function getOutput(thePath) {
+  return fs.readFileSync(thePath, 'utf-8');
+}
+
+// Remove the custom hash from jest-fixture so snapshots are consistent across test runs
+expect.addSnapshotSerializer({
+  print(val) {
+    return val.replace(/jest-fixture-[A-z0-9]*/g, 'jest-fixture');
+  },
+  test(val) {
+    return typeof val === 'string';
+  },
+});
```

Comment:
This is a small snapshot serialiser that just replaces the unique hash appended to jest-fixture directories so that snapshots are consistent across test runs. I don't think it warrants its own file/module/package until we use it in multiple places, was also unsure where to put it.